lambda/aws-qa-appsync-opensearch/question_answering/src/qa_agent/sagemaker_endpoint.py

```diff
@@ -0,0 +1,51 @@
+
+from langchain.llms.sagemaker_endpoint import LLMContentHandler, SagemakerEndpoint
+from aws_lambda_powertools import Logger, Tracer, Metrics
+
+import json
+import os
+logger = Logger(service="QUESTION_ANSWERING")
+
+class ContentHandler(LLMContentHandler):
+    content_type = "application/json"
+    accepts = "application/json"
+
+    def transform_input(self, prompt, model_kwargs) -> bytes:
```

Comment:
this handler is specific to a model ?

Reply:
yes, will open a new issue for sagemaker and bedrock models input/output parsing across models.